source/user-guide/workflow/validation.rst

```diff
@@ -1,5 +1,31 @@
 .. _validation:
 
-Validation
-##########
+Validating records
+##################
 
+In the metadata editor, click the Validate button to trigger the validation.
+
+The right panel display all validation results by levels:
+
+.. figure:: img/validation.png
+
+Click thumbs up or down icon to display errors. Errors reported in blue are for information only and will not affect the global validation of the record:
+
+.. figure:: img/validation-details.png
+
+
+The editor reports also report error messages:
+
+.. figure:: img/validation-inline.png
+
+
+Once a record is validated, the validation status is saved and is displayed in the editor board page:
+
+.. figure:: img/validation-status.png
+
+A facet allows to filter on it. Catalogue administrator can force validation every time and editor leave the metadata editor (see :ref:`system-configuration`).
```

Comment:
Validation status can be filtered using a facet in the editor board. (?)
Catalog administrators can force validation every time an editor leaves the metadata editing interface (so we're not using 'editor' twice to mean two different things)